src/optimizer/join_order/join_relation_set.cpp

```diff
@@ -139,4 +139,32 @@ JoinRelationSet &JoinRelationSetManager::Union(JoinRelationSet &left, JoinRelati
 // 	return GetJoinRelation(std::move(relations), count);
 // }
 
+string GetIndent(int indent) {
+	string result = "";
+	for (int i = 0; i < indent; ++i) {
+		result += "  ";
+	}
+	return result;
+}
+
+static string JoinRelationTreeNodeToString(const JoinRelationTreeNode *node, int indent) {
+	string result = "";
+	string indent_prefix = GetIndent(indent);
+	if (node->relation)
```

Comment:
nit: can you wrap this in curly braces? Just to match the styling of the rest of the code base, we wrap all if statements in curly braces

Reply:
ok ,I changed.